concert/ext/ufo.py

```diff
@@ -545,6 +559,7 @@ async def __ainit__(self, args, average_normalization=True, regions=None, copy_i
         self._producer_condition = asyncio.Condition()
         self._processing_task = None
         self._regions = None
+        self.axis_estimated = kwargs.get("axis_estimated", None)
```

Comment:
why not just a kwarg `axis_estimated`? Now we have kwargs and also **kwargs, which is a weird mixture.